src/Microsoft.AspNetCore.Build/scripts/KoreBuild.ps1

```diff
@@ -150,6 +150,33 @@ if(Test-Path $MSBuildLog) {
     del $MSBuildLog
 }
 
+$MSBuildArguments = @"
+/nologo
+"$Proj"
+/p:KoreBuildToolsPackages="$ToolsDir"
+/p:KoreBuildTargetsPath="$KoreBuildTargetsRoot"
+/p:KoreBuildTasksPath="$MSBuildDir\bin\pub"
+/fl
+/flp:LogFile="$MSBuildLog";Verbosity=diagnostic;Encoding=UTF-8
```

Comment:
Yeah, it is configurable by default using `.\build /v:<verbosity>`. This is for the "ack, everything went wrong and I don't know why" file logger :). It's intentionally mega-verbose so that when a failure occurs, you don't have to re-run at a higher verbosity level to see the results.